### PR TITLE
History mode documentation

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/HNIipHfC

What's changed and why?
Documents what history mode is and the rules that determine if a document is in history mode.

Source document:
https://docs.google.com/document/d/1B2n4DU5YcpxC1a8bsuty1q9lLAroqjBapBOKrhHFAWo/edit#

[Rendered version](https://github.com/alphagov/content-publisher/blob/cdcc03230632b4306b8accba7fbdc5d4620e6c0c/docs/history-mode.md)

### CODE REVIEW

File: docs/history-mode.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+There is an ability to override whether content is defined as political. This is restricted to specific users. These users can specify content is political or not and the previously mentioned criteria will no longer have an affect.
+
+## Who has the ability to change the attributes that affect history mode?
+
+An intention behind history mode is to restrict the ability of publishers to make edits to content once a government has changed. This is to preserve content in the state it was at the point of a government change.
+
+Broadly this is reflected by:
+
+- Only GDS editors have the ability to withdraw, unwithdraw, remove or create new editions of published content that is in history mode.
```
Comment (kevindew) [inline docs/history-mode.md]: I think we should be updated to state which permission the user needs - might need to wait until [Peter's PR](https://github.com/alphagov/content-publisher/pull/1511) is merged to get a link to this in, so could add updating this bit as an AC of his card?

File: docs/history-mode.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+- Managing editors of organisations have the ability to override the political status of content
```
Comment (kevindew) [inline docs/history-mode.md]: Might want to link to the managing editor permission on a user

File: docs/history-mode.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+- Whether it is associated with an organisation that publishes content that mostly represent the policies of the current government
+- Whether the content is tagged to a role appointment (we eventually hope to expand this to be whether the content is tagged to a minister of the respective government)
+
+There is an ability to override whether content is defined as political. This is restricted to specific users. These users can specify content is political or not and the previously mentioned criteria will no longer have an affect.
+
+## Who has the ability to change the attributes that affect history mode?
+
+An intention behind history mode is to restrict the ability of publishers to make edits to content once a government has changed. This is to preserve content in the state it was at the point of a government change.
+
+Broadly this is reflected by:
```
Comment (kevindew) [inline docs/history-mode.md]: Can probably drop the broadly as I think this list is now exclusive.

File: docs/history-mode.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+- Only users with the "manage_live_history_mode" permission can undo the withdrawal
+
+### Removed
+
+- Only users with the "manage_live_history_mode" permission can create a new edition
+
+### Failed to publish
+
+Same as draft.
+
```
Comment (kevindew) [inline docs/history-mode.md]: I don't think you need any of the rest of this doc.

File: docs/history-mode.md (hunk 1 above)
Comment (leenagupte) [reply docs/history-mode.md]: I've spoken with Peter, and updated this paragraph accordingly.

**Comment by leenagupte** [comment]
@kevindew thanks for your comments. I've addressed them and fixed-up

File: docs/history-mode.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+There is an ability to override whether content is defined as political. This is restricted to specific users. These users can specify content is political or not and the previously mentioned criteria will no longer have an affect.
+
+## Who has the ability to change the attributes that affect history mode?
+
+An intention behind history mode is to restrict the ability of publishers to make edits to content once a government has changed. This is to preserve content in the state it was at the point of a government change.
+
+This is reflected by:
+
+- Only Managing editors with the `MANAGE_LIVE_HISTORY_MODE` permission have the ability to withdraw, unwithdraw, remove or create new editions of published content that is in history mode.
```
Comment (kevindew) [inline docs/history-mode.md]: You can remove the create new editions bit here as that's not true and is covered by the next bullet.

**Comment by leenagupte** [comment]
@kevindew Thanks. Have updated and fixed-up.

**Pull request merged**